app.js

```diff
@@ -43,7 +46,28 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(compression());
-app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static(publicDirectory));
+
+if (app.get('env') !== 'development') {
```

Comment:
Isn't Nodemon intended for development, not for production servers? Why is the if condition "not development"? It becomes true in production when `env === "production"`.

Reply:
Yes, you are right. It was a  terrible typo. I'll fix it